modules/hal_nordic/nrfx/Kconfig

```diff
@@ -144,6 +144,7 @@ config NRFX_NVMC
 	depends on $(dt_has_compat,$(DT_COMPAT_NORDIC_NRF51_FLASH_CONTROLLER)) \
 		|| $(dt_has_compat,$(DT_COMPAT_NORDIC_NRF52_FLASH_CONTROLLER)) \
 		|| $(dt_has_compat,$(DT_COMPAT_NORDIC_NRF53_FLASH_CONTROLLER)) \
+		|| $(dt_has_compat,$(DT_COMPAT_NORDIC_RRAM_CONTROLLER))        \
```

Comment:
I believe this is wrong. Devices with the RRAM controller should use the RRAMC driver (`nrfx_rramc.c`), not the NVMC one (`nrfx_nvmc.c`), so a new option for that driver should be added instead. Or am I missing something?

Reply:
AFIK settings of this option adds nrf_nvmc.c nrfx driver to the build.